.env.development

```diff
@@ -23,4 +23,4 @@ AUTHN_MINIMAL_HEADER=true
 LOGIN_ISSUE_SUPPORT_LINK=''
 TOS_AND_HONOR_CODE='http://localhost:18000/honor'
 PRIVACY_POLICY='http://localhost:18000/privacy'
-REGISTRATION_OPTIONAL_FIELDS='gender,goals,level_of_education,year_of_birth'
+REGISTRATION_OPTIONAL_FIELDS='{"gender":true,"goals":true,"levelOfEducation":true,"yearOfBirth":true}'
```

Comment:
This works locally but not sure if it will with `frontend_build` script in tubular. We had a problem with this variable in the past with space-separated values.

Reply:
Instead of using a true/false, can’t we simply remove the values from `REGISTRATION_OPTIONAL_FIELDS` array to hide them from the form? If there is no field specified in it, you could simply hide all the content related to optional fields.

Reply:
yes it can work, I will update it